sources/packages/api/src/utilities/msfaa-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { OfferingIntensity } from "../database/entities";
+
+/**
+ * Gets the marital status code conversion for the MSFAA request file
+ */
+export function getMaritalStatusCode(maritalStatus: string): string {
+  if (maritalStatus === "married") {
+    return "M";
+  }
```

Comment:
this can also be simplified if you would like to, return maritalStatus === "married" ? "M" : maritalStatus === "single" ? "S" : "O";

Reply:
I tried it and code smells picked it and said "Ternary operators should not be nested". So changed it.
https://rules.sonarsource.com/javascript/RSPEC-3358

Reply:
IMO if there is more that 2 marital status possible, switch case would be nice.

Reply:
> IMO if there is more that 2 marital status possible, switch case would be nice.

😄 it was exactly what I also mentioned in a call.